charts/newrelic-infrastructure/README.md

```diff
@@ -166,6 +166,7 @@ integrations that you have configured.
 | kubelet.extraVolumeMounts | list | `[]` | Defines where to mount volumes specified with `extraVolumes` |
 | kubelet.extraVolumes | list | `[]` | Volumes to mount in the containers |
 | kubelet.hostNetwork | bool | Not set | Sets pod's hostNetwork. When set bypasses global/common variable  |
+| kubelet.scraperMaxReruns | int | 4 | Max number of kubelet scraper reruns when scraper runtime error happens. |
```

Comment:
this should be `kubelet.config.scraperMaxReruns` also move this to line 163 next to the other `kubelet.config` params.

Reply:
moved